packages/grid/x-data-grid-pro/src/tests/reorder.DataGridPro.test.tsx

```diff
@@ -79,265 +80,363 @@ describe('<DataGridPro /> - Reorder', () => {
       await raf();
       expect(getColumnHeadersTextContent()).to.deep.equal(['brand', 'id']);
     });
```

Comment:
No need to do now but this file could be split into one for row reordering and another for column reordering.

Reply:
I spit the 2 files ;)